src/samVisualize/rendering/navigation.ts

```diff
@@ -0,0 +1,45 @@
+/*!
```

Comment:
is the rendering/ subdirectory really necessary?

Reply:
There will be a few other functionalities in that directory, which I will add in the next PR.